### PR TITLE
Run CI regularly twice a month

### LINKED ISSUE #549: Consider Cron jobs in CI to help maintenance
Setting up a cron job to run once a week / month would be useful to check the status of SR + dependencies

### PR DESCRIPTION
**Pull request by peterzahemszky**
Run CI regularly twice a month to discover dependency issues (like #574) earlier. Currently they're arbitrarily scheduled for the 7th and 22nd days of the month.

From the documentation of [`on.schedule`](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onschedule):
> Scheduled workflows run on the latest commit on the default or base branch.

Closes #549

### CODE REVIEW

File: .github/workflows/build_centos7.yml [hunk 1]
```diff
@@ -63,4 +66,4 @@ jobs:
           . venv/bin/activate
           export LC_ALL=en_US.UTF-8
           export LANG=en_US.UTF-8
-          make pythontest
\ No newline at end of file
+          make pythontest
```
Comment (peterzahemszky) [inline .github/workflows/build_centos7.yml:69]: This extra newline character at the end of the file is unrelated to the rest of the PR

**Comment by flongford** [comment]
FYI - merging this PR will also close #549

**Review by flongford — COMMENTED** [review]
Suggestion: change `name` on each workflow to include OS

**Review by flongford — APPROVED** [review]
LGTM!
